packages/embed/src/utils/load-options-from-attributes.ts

```diff
@@ -25,6 +25,11 @@ const transformInteger = (value: string | null): number | undefined => {
   return isNaN(integer) ? undefined : integer
 }
 
+const transformFunction = (value: string | null): Function | undefined => {
+  const fn = value && value in window ? window[value] : undefined
```

Comment:
Does this work? When I played with this I had to do `window[value].call()`.

Reply:
isn't it supposed to return a function, instead of calling it? If so - yep, it works, also there's a unit test you can check out

Reply:
You are right. In that case should it return `window[value].call`? The `window[value]` is not callable with just `()`.

Reply:
Why do you think it shouldn't work?

Reply:
also, .call signature is not the same as the function's signature

Reply:
![image](https://user-images.githubusercontent.com/5614659/108715192-30534200-7523-11eb-8271-4d097ee947e3.png)

For instance, it works with `window.getSelection`. Or you have some edge case in mind?

Reply:
I double checked and now it works for me too :) 👍